README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
```

Comment:
I believe the `install_requires` in setup and `requirements.txt` are used for sightly different contexts. `install_requires` should contain urgent dependencies which are required for the build. While `requirements.txt` should be exhaustive of all the dependencies. For example, we can put numpy in setup, while matplotlib in `requirements.txt`